pkg/tsdb/cloudwatch/cloudwatch_query.go

```diff
@@ -21,6 +21,7 @@ type cloudWatchQuery struct {
 	Dimensions       map[string][]string
 	Period           int
 	Alias            string
+	Label            *string
```

Comment:
Does this need to be a pointer now? If feature is enabled and string is not empty, we always add label to the metric data query request, no? Should be no need for the `nil` state or am I missing something?